src/components/common/SearchPannel/searchPannel.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../assets/styles/variables.scss';
+
+.search{
+    &-container{
+        padding: 1rem;
+        display: flex;
+        align-items: center;
+        width: 26.25rem;
+        border: none;
+        border-radius: 0.6rem;
+        overflow: hidden;
+        background-color: $light-color;
+    }
+    &-button{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 0;
+        border: none;
+        background-color: transparent;
+        width: 1rem;
+        height: 1.06rem;
+        
+        &:hover {
+            cursor: pointer;
```

Comment:
Button element has `cursor: pointer` by default

Reply:
Not at all.